ecommerce/serializers.py

```diff
@@ -367,15 +367,25 @@ def _update_items(cls, basket, items):
             item = items[0]
             product_id = item.get("product_id")
             run_ids = item.get("run_ids")
-            product = models.Product.objects.get(id=product_id)
+            if isinstance(product_id, int) or product_id.isdigit():
```

Comment:
Also I believe, product_id form query param would either be `string-integer` or `string-readable id`. 
so why need for `isinstance(product_id, int)` ?

Reply:
product_id can be either int or string. If we don't use `isinstance(product_id, int)`, it would fail if product_id is integer.
1. isinstance(product_id, int) => int  (product_id) it being used in test cases and we are passing it in almost 18 test cases.
2. product_id.isdigit() is for string-integer or string-readable id

Reply:
If you still want to update all the test cases then let me know, I will do that.

Reply:
Try to run all the test cases after removing `isinstance(product_id, int)`, you will get it.